docs/sql-ref-syntax-aux-conf-mgmt-set-timezone.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: global
+title: SET TIME ZONE
+displayTitle: SET TIME ZONE
+license: |
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+ 
+     http://www.apache.org/licenses/LICENSE-2.0
+ 
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+---
+
+### Description
+
+The `SET TIME ZONE` command sets the current default time zone(`spark.sql.session.timeZone`) displacement for the `SparkSession`. 
+
+### Syntax
+
+```sql
+SET TIME ZONE LOCAL
+SET TIME ZONE 'timezone_value'
+SET TIME ZONE INTERVAL interval_literal
+```
+
+### Parameters
+
+* **LOCAL**
+
+    Respectively set the time zone the one specified in environment variable `TZ`, or to the operating system time zone if `TZ` is undefined.
```

Comment:
set the time zone "to" the one specified

Reply:
`operating system` -> `system`? We actually use the JVM timezone which can be different from the operating system,

Reply:
yeah, it prefers `user.timezone`